presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/GeometryUtils.java

```diff
@@ -34,13 +50,53 @@
 import static com.esri.core.geometry.GeometryEngine.geometryToEsriShape;
 import static com.esri.core.geometry.OperatorImportFromESRIShape.local;
 import static com.esri.core.geometry.ogc.OGCGeometry.createFromEsriGeometry;
+import static io.airlift.slice.SizeOf.sizeOfByteArray;
+import static io.airlift.slice.SizeOf.sizeOfDoubleArray;
+import static io.airlift.slice.SizeOf.sizeOfIntArray;
+import static java.lang.Math.toIntExact;
 import static java.nio.ByteOrder.LITTLE_ENDIAN;
 import static java.util.Collections.emptyList;
 
 public final class GeometryUtils
 {
     private static final int POINT_TYPE = 1;
 
+    private static final int VERTEX_DESCRIPTOR_INSTANCE_SIZE = ClassLayout.parseClass(VertexDescription.class).instanceSize() +
+            toIntExact(sizeOfDoubleArray(2) + sizeOfIntArray(21));
+    private static final int MULTI_POINT_IMPL_INSTANCE_SIZE = 56;
+    private static final int MULTI_PATH_IMPL_INSTANCE_SIZE = 112;
```

Comment:
Where do these numbers come from?

Reply:
These are the values of ClassLayout.parseClass(XXX.class).instanceSize() for MultiPointImpl, MultiPathImpl, etc. classes which are package private in com.esri.core.geometry. As you mentioned in the other comment, the result of ClassLayout.parseClass(XXX.class).instanceSize() may change. Should I add a helper class in com.esri.core.geometry package to compute these numbers instead of hard-coding them? Is there a better way?